business_requirement_deliverable_cost/models/business.py

```diff
@@ -123,6 +123,24 @@ class BusinessRequirementDeliverable(models.Model):
         groups='business_requirement_deliverable_cost.'
         'group_business_requirement_estimation',
     )
+    gross_profit = fields.Float(
+        compute='_compute_gross_profit',
+        string='Gross Profit',
+    )
+
+    @api.multi
+    @api.depends('resource_ids.price_total',
```

Comment:
I am not sure this does what I am expecting.
Here is what I need: DL.gross_profit = DL.price_total - sum(RL.cost)
It should be updated whenever the RL.cost or DL.price_total changes.

Can you double check? @sudhir-serpentcs